input/fsh/au-erequesting-path-request.fsh

```diff
@@ -0,0 +1,23 @@
+Profile: AUeRequestingPathologyServiceRequest
+Parent: AUeRequestingServiceRequest
+Id: au-erequesting-path-request
+Title: "AU eRequesting Pathology ServiceRequest"
```

Comment:
Remove the word "Service" from the title.

Reply:
Done